apps/teamsjs-diagnostic-app/src/components/sample/Scenario1.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from 'react';
+import './Scenario1.css';
+import { captureConsoleLogs } from './LoggerUtility';
+import { app } from '@microsoft/teams-js';
+import { registerOnResume } from '../../apis/AppApi';
+import { authenticateUser } from '../../apis/AuthenticationStart';
+import apiComponents from './ApiComponents';
+
+type Log = string;
+
+interface Scenario1Props {
+  showFunction?: boolean;
+}
+
+export function Scenario1({ showFunction }: Scenario1Props) {
+  const [logStatements, setLogStatements] = useState<Log[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+
+  useEffect(() => {
+    const filteredLogs: Log[] = [];
+    captureConsoleLogs((log: string) => {
+      if (!log.includes('Get basic user info from SSO token')) {
```

Comment:
Filtering based on strings can lead to various security concerns what if there is no such text but token is present would this logic still work?